core/modules/system/system.install

```diff
@@ -3137,6 +3137,15 @@ function system_update_1072() {
   // Empty update to force a rebuild of hook_library() and JS aggregates.
 }
 
+/**
+ * Add default values for message settings.
+ */
+function system_update_1073() {
+  $config = config('system.core');
+  $config->set('messages_dismissable', 0);
+  $config->save();
+}
+
```

Comment:
Not sure if the update hook is required here. If not set, `config_get()` will return `NULL`, which would work fine with the conditionals that load the js library.

Reply:
Yes, but I believe there may be warnings/errors displayed or in the log as a result... Better safe than sorry I think.